module/metrics/ping.go

```diff
@@ -40,4 +48,11 @@ func (pc *PingCollector) NodeReachable(node *flow.Identity, nodeInfo string, rtt
 		LabelNodeInfo:    nodeInfo,
 		LabelNodeVersion: version}).
 		Set(rttValue)
+
+	pc.sealedHeight.With(prometheus.Labels{
+		LabelNodeID:      node.NodeID.String(),
+		LabelNodeAddress: node.Address,
+		LabelNodeRole:    node.Role.String(),
+		LabelNodeInfo:    nodeInfo}).
+		Set(float64(sealedHeight))
```

Comment:
since we have the info, maybe we can include the node version for this metric as well.

Reply:
added node version to the metric.